web/styles/subscriptions.css

```diff
@@ -59,26 +59,30 @@
     margin-right: 10px;
 }
 
+#add_new_user_group {
+    height: 2em;
+}
+
 .create_user_group_plus_button,
 .create_stream_plus_button {
-    font-size: 24px;
     font-weight: 400;
     position: relative;
-    top: 2px;
     border: 1px solid hsl(0deg 0% 80%);
     border-radius: 5px;
     background-color: hsl(0deg 0% 100%);
     color: hsl(0deg 0% 0%);
     margin: 0 0 0 5px;
-    height: 27px;
+    height: 100%;
+    display: block;
 
     &:hover {
         border: 1px solid hsl(0deg 0% 47%);
     }
 
     .create_button_plus_sign {
-        position: relative;
-        top: -5px;
+        line-height: 1em;
+        display: block;
```

Comment:
This is a flex item, so they display: block property here is unnecessary. I also couldn't see any difference when switching off the `line-height` here either, though if it needs to stay, make it unitless.

Reply:
Oh wait...it looks like a bunch of this gets overwritten in the next commit?

Reply:
Ah yeah, I wrote this commit before changing the icon. Maybe it makes sense to combine them into one commit now?

Reply:
Probably, yeah, if that's not too much work. It'll make the history cleaner here.

Reply:
It's not much work at all! Just squashing two commits, very simple. Done :)